spec/factories/user_factory.rb

```diff
@@ -7,7 +7,7 @@
     email { "someone@example.com" }
 
     trait :managing_editor do
-      permissions { [User::MANAGING_EDITOR_PERMISSION] }
+      permissions { [User::MANAGING_EDITOR_PERMISSION, User::PRE_RELEASE_FEATURES_PERMISSION] }
```

Comment:
I think @1pretz1 is likely to do something a bit more with permissions so this may change - this is just a heads up.